packages/perseus/src/widgets/explanation.tsx

```diff
@@ -110,53 +111,15 @@ class Explanation extends React.Component<Props, State> {
                 </div>
             );
         } else {
-            if (isArticle) {
-                // NOTE: For articles, the baseElements `Link` may is either be
-                // an anchor tag (if rendered in a modal) or a Wonder Blocks
-                // link component in lesson page, see: `article-page.jsx`.
-                linkContainer = (
-                    <div
-                        className={css(
-                            styles.linkContainer,
-                            styles.articleLink,
-                        )}
-                    >
-                        <Link
-                            className={css(styles.explanationLink)}
-                            // HACK(michaelpolyak): WB Link doesn't passthrough
-                            // class name.
-                            style={styles.explanationLink}
-                            href={href}
-                            onClick={onClick}
-                            role="button"
-                            aria-expanded={this.state.expanded}
-                        >
-                            {`[${linkAnchor}]`}
-                        </Link>
-                    </div>
-                );
-            } else {
-                // NOTE: For exercises, the baseElements `Link` is an
-                // anchor tag, see: `perseus-api.jsx`.
-                linkContainer = (
-                    <div
-                        className={css(
-                            styles.linkContainer,
-                            styles.exerciseLink,
-                        )}
-                    >
-                        <Link
-                            className={css(styles.explanationLink)}
```

Comment:
What did these styles do? (`linkContainer`, `exerciseLink`, `articleLink`, and `explanationLink`)

Reply:
`explanationLink` provides color and italics.
`exerciseLink` and `articleLink` provide font size and line height.
I don't know what value `linkContainer` provides. I think originally it was there to convert the containing `<div>` tag to an inline element. Now that the container is replaced with a `<button>`, there is no need for that styling. I will remove it (testing shows that doing so does not impact usage).